src/main/java/software/amazon/awssdk/crt/internal/GraalVMNativeFeature.java

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+package software.amazon.awssdk.crt.internal;
+
+import org.graalvm.nativeimage.hosted.Feature;
+
+import software.amazon.awssdk.crt.CRT;
+
+/**
+ * Implementation of GraalVM feature to extract the share lib to the image path.
+ * Internal API, not for external usage.
+ */
+public class GraalVMNativeFeature implements Feature {
+
+    @Override
+    public void afterImageWrite(AfterImageWriteAccess access) {
+        new CRT();
```

Comment:
Is it OK that we're running CRT (extracting the lib to /tmp and loading it etc...) during the build process?

If it's problematic, maybe we could we shuffle code around so that whatever helper functions this needs to call, don't actually rely on the native code

Reply:
why it will be problematic?